### DISCUSSION TITLE
关于绳网的一点想法。

### DISCUSSION THREAD

**Discussion by HifumiMizuhara** [share121/inter-knot#2290] [其他]
看绳网用Vue框架写的，可以考虑用Electron封装成应用程序，这样或许可以省去装插件的麻烦（可以直接Electron内核禁用CORS拦截），也让手机版用户用的更舒心。
当然本人不是大佬，只是提一个小想法。

**Comment by qishaming-mihomo** [comment]
完全不懂，但是顶一下先

**Comment by wssjl** [comment]
感觉现在差个点赞功能，我都不能给你点赞

**Comment by Robei1** [comment]
Electron的内存泄露比较严重，要优化才可以
